src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,48 @@
+package homework_2.pyramid_printer;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+@Getter
+@Setter
+class PyramidPrinter {
```

Comment:
Should be public. Here and in other places: please fix access modifiers

Reply:
Checked and fixed in all classes, including tests.